charts/metallb/templates/controller.yaml

```diff
@@ -36,6 +36,7 @@ spec:
       securityContext:
         runAsNonRoot: true
         runAsUser: 65534 # nobody
+        fsGroup: 65534
```

Comment:
This will need to go in manifests/metallb.yaml too, right?

Reply:
thanks, I fixed that